docs/readme.md

```diff
@@ -101,7 +101,8 @@ Currently, the features supported are:
 | `rover`               | Deploy SkyWalking Rover and detect the processes in the Kubernetes environment.                                                            | Only support deployment in the Kubernetes environment, docker is not supported.                                                       |
 | `mysql`               | Start a MySQL server and load generator to execute the sample SQLs periodically, set up fluent bit to fetch slow logs and export to OAP, and export their metrics to SkyWalking.                                        |                                                                                                                                       |
 | `postgresql`          | Start a PostgreSQL server, and load generator to execute the sample SQLs periodically, set up fluent bit to fetch slow logs and export to OAP, and export their metrics to SkyWalking.                                  |                                                                                                                                       |
-| `apisix-monitor`          | Deploy OpenTelemetry and export APISIX metrics to SkyWalking for analysis and display on UI                               |                                                                                                                                       |
+| `apisix-monitor`      | Deploy OpenTelemetry and export APISIX metrics to SkyWalking for analysis and display on UI                               |                                                                                                                                       |
+| `mesh-with-agent`     | Deploy services with java agent in the service mesh environment. | OnlySupport deployment in the Kubernetes environment, docker is not supported.|
```

Comment:
So, we are going to use this in the demo?

What does this look like? Could you share?

Reply:
> So, we are going to use this in the demo?
Yes.
>What does this look like? Could you share?
It's generally same as the current demo environment, you could try on this address: http://34.81.178.71:8080/
I have replaced the `java-agent-injector` and `als` features with the `mesh-with-agent`.

Reply:
Could we update the group name in the mesh from `agentless` to `mesh-svr`?

Reply:
I have updated the config and test address, please take a look. :)